app/services/catalog/update_order_item.rb

```diff
@@ -0,0 +1,58 @@
+module Catalog
+  class UpdateOrderItem
+    class OrderItemNotFound < StandardError; end
+    class ServiceInstanceNotFound < StandardError; end
```

Comment:
Doesn't look like this exception is used anywhere else, can it be removed?

Reply:
Ah, oops. This needs to be totally changed to a different exception.

Reply:
@lindgrenj6 Can you take another look? This time the exception isn't for when the service instance isn't found (which I think would result in a different error which would bubble up to the `service_order_listener`), but if for some reason the `external_url` isn't available on the service_instance.